### PR TITLE
Wfp 652

### PR DESCRIPTION
**Pull request by carlov20**

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsallocations/service/UnallocatedCasesService.kt [hunk 1]
```diff
@@ -22,4 +22,9 @@ class UnallocatedCasesService(
     val convictions = communityApiClient.getConvictions(crn)
     return convictions.sortedByDescending { c -> c.convictionDate }.first().sentence.startDate
   }
+
+  fun getInitialAppointmentDate(crn: String, contactsFromDate: LocalDate): LocalDate? {
+    val contacts = communityApiClient.getInductionContacts(crn, contactsFromDate)
+    return contacts.minByOrNull { c -> c.contactStart }?.contactStart?.toLocalDate()
```
Comment (carlov20) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsallocations/service/UnallocatedCasesService.kt:28]: should we store the time and convert it to localDate when retrieving or follow what I did here? i.e. convert from the delius response and only care about the date?

Comment (carlov20) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsallocations/service/UnallocatedCasesService.kt:28]: Just checked the prototype and we always only ever care about the date so I think we are good to drop the time here and store only the date

File: helm_deploy/hmpps-allocations/values.yaml [hunk 2]
```diff
@@ -39,8 +39,8 @@ generic-service:
   namespace_secrets:
     hmpps-allocations:
       APPINSIGHTS_INSTRUMENTATIONKEY: "APPINSIGHTS_INSTRUMENTATIONKEY"
-      AUTH_API_CLIENT_ID: "OAUTH_CLIENT_ID"
-      AUTH_API_CLIENT_SECRET: "OAUTH_CLIENT_SECRET"
+      OAUTH_CLIENT_ID: "AUTH_API_CLIENT_ID"
+      OAUTH_CLIENT_SECRET: "AUTH_API_CLIENT_SECRET"
```
Comment (markreesmoj) [inline helm_deploy/hmpps-allocations/values.yaml:43]: this is so confusing!

**Review by markreesmoj — APPROVED** [review]
🔥

**Pull request merged**